### ISSUE TITLE
Color Mapping module crashes Darktable 3.4

### ISSUE THREAD

**Issue by Dave22152** [darktable-org/darktable#8069]
Issue: Darktable crashes when applying the Color Module 

Version and system info: DT 3.4 for Windows 10.  HP Pavilion with AMD A8-7810 APU and AMD Radeon R5 Graphics, 8.0 GB RAM, 64-bit OS and x64 based processor.  Raw files are Canon CR2

Steps to replicate problem:  When editing a raw file, invoke the Color Mapping module.  Select "Acquire Source" and open destination file.  Invoke Color Mapping and select "Acquire Target".  The system processes and once the operation is complete and the color is mapped, the software immediately crashes.  The module history is not updated and there is no backtrace.

This happens with all of my raw files.  I realize that there isn't much to go on without a backtrace, but that is all I have.  I will be happy to provide your more information if at all possible

Thanks for your attention.

Dave G.

**Comment by ralfbrown** [comment]
Likely a duplicate of #7879, which has been fixed in master.

**Comment by vkontog** [comment]
Just installed the 3.4.1 and tried the color mapping module. I 've create a preset from another pic and when i pressed the "acquire target", darktable just shut down.

My OS is windows 10, Intel(R) Core(TM) i5-4460 CPU  and GPU AMD Radeon R9 200 Series.

**Comment by Mark-64** [comment]
This is a known issue. fixed in master, but unfortunately the fix is not in 3.4.1.1

**Comment by Dave22152** [comment]
Verified to be fixed.  Closing

**Comment by prabhath-mp** [comment]
I have just reported a similar incident (sorry couldn't find this thread earlier)
Has someone found a solution to this, yet?

**Comment by Dave22152** [comment]
This issue was resolved in 3.5, so you should see the correction when they release 3.6 this July.

**Comment by prabhath-mp** [comment]
Hi All,

Apologies for the delayed response.

Thanks for all your help in resolving this problem.

I have been successfully using the application 👍 

Cheers from Bangalore, India! 😃
